docker/README.md

```diff
@@ -57,7 +57,7 @@ Using the invite code from your guardian dashboard, join the federation using `f
 docker run -it --rm \
   -e RUST_LOG=off \
   -v "$(pwd)/mutinynet-client":/mutinynet-client \
-  fedimint/fedimint-cli:v0.7.0 \
+  fedimint/fedimint-cli:0.7.0 \
```

Comment:
Why would the naming of old releases change? I'd just publish with the new convention from now on and then switch in docker compose when we update the container versions.